distribution/docs/src/main/jdocs/content/_plugins/metacard-attribute-security-policy-plugin-contents.adoc

```diff
@@ -0,0 +1,26 @@
+:type: plugin
+:status: published
+:title: Metacard Attribute Security Policy Plugin
+:link: _metacard_attribute_security_policy_plugin
+:plugintypes: policy
+:summary: Collects attributes into a security field for the metacard.
+
+The *Metacard Attribute Security Policy Plugin* collects attributes from a metacard and combine these attributes into a security field for the metacard.
+
+This plugin assumes that the pertinent information has already been parsed out of the metadata and placed directly on the metacard itself.
+The plugin supports mapping attributes from their names on the metacard to a different security policy name. Attributes can be mapped by union or intersection. E.g.
```

Comment:
❓ All this really says is that a union produces a union and an intersection produces an intersection. Should we have some actual example values to demonstrate? I don't think this is adding much value as is. The usage detail of `src=dest` should be documented separately and explicitly.